doc/getting-started.md

```diff
@@ -0,0 +1,61 @@
+# Getting Started with GTT SMASH plugin
+
+The **Redmine GTT SMASH** plugin allows you to connect your [Redmine GTT](https://gtt-project.org/) projects with the [SMASH mobile app](https://www.geopaparazzi.org/smash/index.html) to create field notes and upload them as issues to [Redmine](https://redmine.org/).
+
+SMASH is a digital field mapping application developed to perform fast qualitative engineering/geologic surveys and GIS data collection.
+
+## Installing SMASH
+
+You can install the SMASH app from the app stores:
+
+* *SMASH Digital field mapping* for iOS devices on the [Apple store](https://apps.apple.com/us/app/smash-digital-field-mapping/id1475079936).
+* *SMASH field mapping* for Android devices on the [Google Play store](https://play.google.com/store/apps/details?id=eu.hydrologis.smash&hl=en).
+
+The following plugin documentation gives a brief introduction in how to use SMASH with Redmine.
+
+SMASH provides additional functionality that is available in the [SMASH User Manual](https://www.geopaparazzi.org/smash/index.html).
+
+## Integration in Redmine GTT
+
+To get projects connected with SMASH mobile application make sure to enable the **GTT SMASH** module in your Redmine project settings.
+
+![Project Modules](project_modules.png)
+
+## Connect SMASH mobile application to Redmine
+
+### Connect Redmine account
+
+In SMASH mobile application top menu go to **Export** or **Import**. Click on the gear icon of **GTT** and you will see the following:
+
+![GTT URL](import_setting.png) ![GTT URL](gtt_url.png)
+
+* **Server URL**: enter your Redmine URL `example.com`
+* **Server Username**: your Redmine account's `username`
+* **Server Password**: your Redmine account's `password`
+
+### Import Redmine projects
+
+In menu again go to **Import** &#8594; **GTT** &#8594; Select Project &#8594; Click **Import**
+
+![GTT Import](gtt_import.png)
+
+This will import your project's form configuration to mobile storage.
```

Comment:
@dkastl @nobnisai (CC: @mbasa-georepublic)
To enable downloaded form (`gtt_tags.json`), restarting SMASH app was necessary at last year's SMASH version.
If it is still necessary, maybe better to mention that.

Reply:
@sanak Only the new import is needed. We don't need to restart the app.

Reply:
@nobnisai Okay, thanks for confirmation!
Then, I resolve this conversation.